src/bulk/common.ts

```diff
@@ -346,16 +344,21 @@ export class WriteConcernError {
   }
 
   /** Write concern error message. */
-  get errmsg(): string {
+  get errmsg(): string | undefined {
     return this.err.errmsg;
   }
 
-  toJSON(): { code?: number; errmsg: string } {
-    return { code: this.err.code, errmsg: this.err.errmsg };
+  /** Write concern error info. */
+  get errInfo(): Document | undefined {
```

Comment:
Why did we add this new getter again? There is spec related to this I believe, this makes our WriteConcernError "more" compliant/consistent.

Reply:
I didn't have any specific reason, just consistency between the getters of other props.